packages/transaction-messages/tsconfig.json

```
@@ -0,0 +1,9 @@
+{
+    "$schema": "https://json.schemastore.org/tsconfig",
+    "compilerOptions": {
+        "lib": ["DOM", "ES2017", "ES2022.Error"]
+    },
+    "display": "@solana/transactions",
+    "extends": "@solana/tsconfig/base.json",
```

I screwed you in #2355.
```suggestion
    "extends": "../tsconfig/base.json",
```